### PR TITLE
on click make item type text/author-text foucsed with higher z-index

### PR DESCRIPTION
**Pull request by igor-kamil**

### CODE REVIEW

File: resources/views/topics/show.blade.php [hunk 1]
```diff
@@ -83,6 +83,15 @@ class="item"
                     fixedContentPos: false
             });
         });
+
+        $(document).click(function(event) {
```
Comment (eronisko) [inline resources/views/topics/show.blade.php]: This won't be present in the visual editor (not sure if that's intended)

File: resources/views/topics/show.blade.php [hunk 2]
```diff
@@ -83,6 +83,15 @@ class="item"
...
+
+            $('.item-container').removeClass('focused');
+
+            if ($(event.target).is('.item-author-text, .item-text') || ($(event.target).parents('.item-author-text, .item-text').length > 0)) {
```
Comment (martin-havala) [inline resources/views/topics/show.blade.php]: should not it be for general item-container?

File: resources/views/topics/show.blade.php (hunk 1 above)
Comment (igor-kamil) [reply resources/views/topics/show.blade.php]: intended...

File: resources/views/topics/show.blade.php (hunk 2 above)
Comment (igor-kamil) [reply resources/views/topics/show.blade.php]: I was thinking about the same... maybe later can be applied general for `'.item'`

**Pull request merged**